src/utils/meta.tsx

```diff
@@ -1,39 +1,46 @@
 import * as React from 'react'
-import { SeoContent, Story, WithStory } from '../storyblok/StoryContainer'
+import { SeoContent, Story } from '../storyblok/StoryContainer'
 import { getPublicHost, getStoryblokImage } from './storyblok'
 
-export const getMeta = ({
-  story,
-  nonce,
-}: WithStory<Story & { content: SeoContent }> & { nonce?: string }) => (
+interface Meta {
+  story?: Story & { content: SeoContent }
+  nonce?: string
+  fullSlug?: string
+  title?: string
+}
+
+const getFullSlugFromStory = (story?: Story) =>
+  story && story.full_slug.replace(/\/?home$/, '')
+
+export const getMeta = ({ story, title, nonce, fullSlug }: Meta) => (
   <>
-    <title>{story.name}</title>
+    <title>{title ? title : story && story.name}</title>
     <link
       rel="canonical"
-      href={`${getPublicHost()}/${story.full_slug.replace(/\/?home$/, '')}`}
+      href={`${getPublicHost()}/${fullSlug || getFullSlugFromStory(story)}`}
     />
-    {story.content.robots && (
+    {story && story.content.robots && (
```

Comment:
Doesn't this code hurt you a little? :wink: I feel like there's probably a good ramda-style function for doing this 🙂